tcms/testruns/templates/testruns/get.html

```diff
@@ -282,15 +282,49 @@ <h5><span class="test-executions-count"></span> Results</h5>
                                             </div>
                                         </div>
                                         <div class="col-md-3">
-                                            <span>{% trans 'Test Case Attachments' %}:</span>
-                                            <ul class="list-group list-unstyled test-case-attachments">
-                                                <template id="attachments-list-item">
-                                                    <li class="list-group-item'">
-                                                        <a></a>
-                                                    </li>
-                                                </template>
-                                                <li class="list-group-item hidden">{% trans "No attachments" %}</li>
-                                            </ul>
+                                            <div>
```

Comment:
IMO this is better to become SimpleMDE because it will bring more improvements. Also related to that the comment form + status buttons seems to small in col-3. Let's use col-6 for the text & info + col-6 for comments & attachments.

Note: see core.widgets.SimpleMDE

Warning: we've never used this multiple times in the same page so not sure how it will work. We do need some special care for uploaded images/files but that's a separate issue.

You probably need to split this PR into smaller chunks.

Reply:
> You probably need to split this PR into smaller chunks.

I can remove the kebab button and delete func from this PR, but the form + visualization (loading of comments via the API) should stay in one place so that we can test the changes